src/main/java/org/folio/circulation/domain/OpeningDay.java

```diff
@@ -92,11 +113,16 @@ private JsonArray openingHourToJsonArray() {
   }
 
   public JsonObject toJson() {
-    DateTime dateTime = date.toDateTime(LocalTime.MIDNIGHT, DateTimeZone.UTC);
-    return new JsonObject()
-      .put(DATE_KEY, DATE_TIME_FORMATTER.print(dateTime))
+    JsonObject json = new JsonObject()
       .put(ALL_DAY_KEY, allDay)
       .put(OPEN_KEY, open)
+      .put(EXCEPTIONAL_KEY, exceptional)
       .put(OPENING_HOUR_KEY, openingHourToJsonArray());
+
+    if (date != null) {
+      DateTime dateTime = date.toDateTime(LocalTime.MIDNIGHT, DateTimeZone.UTC);
```

Comment:
Why is the date converted to midnight during serialization?

Reply:
Because `mod-calendar` returns date in ISO8601 with time set to zeroes: 
`"date": "2020-01-27T00:00:00.000+0000"`

Reply:
That suggests that this code is also used for faking responses from the `calendar` interface. If so, I think that this should be done in testing code, not production code (as with other comments).